File: Mix/edabit Easy/EdabitEasyChallenges/src/HowMuchIsTrue.java
public class HowMuchIsTrue {
    public static int countTrue(boolean[] arr) {
		int arrayAcumulator = 0;
		for(int i=0; i < arr.length; i++){
			if(arr[i] == true){
				arrayAcumulator += 1;
			}
		}
		return arrayAcumulator;
	}

    public static void main(String[] args) {
        HowMuchIsTrue obj = new HowMuchIsTrue();
        boolean a[] = {false, false, false};
        System.out.println(obj.countTrue(a));
    }   
}

File: Mix/Edabit/HowMuchisTrue.js
//Platform: Edabit
//Dificult: Medium
//Challenge's name: How Much is True?
//Link: https://edabit.com/challenge/GLbuMfTtDWwDv2F73


//Create a function which returns the number of true values there are in an array.

function countTrue(arr) {
    let sum =0; 
    let trues = arr.filter(element => element == true);
    return trues.length;
}